### DISCUSSION TITLE
I want to using cantonese but the choise only have one for chinese?how can I use?

### DISCUSSION THREAD

**Discussion by cantoneseslang** [openai/whisper#1776] [Q&A]
I want to using cantonese but the choise only have one for chinese?how can I use?and also in the app,same question of the using cantonese its un-selectable

**Comment by BackMountainDevil** [comment]
jesus. cantonese is one of Chinese. zh-cn, zh-hk, zh-tw.... search before ask https://github.com/openai/whisper/discussions/277
